holidays/examples/vacanza.py

```
@@ -0,0 +1,85 @@
+        help="Use 'public' for only public holidays",
+    )
+
+    args = parser.parse_args()
+
+    try:
+        years = parse_years(args.years)
+        public_only = args.type == "public"
+
+        generate_ics(args.country, years, public_only)
+
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)